### ISSUE TITLE
Publish to AWS ECR Public Gallery

### ISSUE THREAD

**Issue by wuhkuh** [renovatebot/docker-renovate#240]
Hey! Is it possible to also publish the renovate images to the [AWS ECR Public Gallery](https://gallery.ecr.aws/)?
Docker Hub may be too restrictive in terms of usage quota at times. This would save us (and probably others, too) mirroring renovate images to internal registries 😄

**Comment by viceice** [comment]
currently not. but some images will be available on ghcr.io soon

**Comment by JamieMagee** [comment]
Renovate images are available on Google's public Docker Hub mirror

```sh
$ docker pull mirror.gcr.io/renovate/renovate:latest
latest: Pulling from renovate/renovate
Digest: sha256:efff49102c74aec8fdd56a8405676fd16b6c02804813555c1b9306e7a0f26f50
Status: Image is up to date for mirror.gcr.io/renovate/renovate:latest
mirror.gcr.io/renovate/renovate:latest
```

It's not perfect if you're trying to co-locate Renovate images near your docker hosts, but doesn't have any rate limits like Docker Hub.

**Comment by dc2tom** [comment]
There appears to be an entry in the ECR public gallery but no image tags - oddly there is a count of 3.5k+ downloads 

https://gallery.ecr.aws/renovate/renovate

**Comment by viceice** [comment]
Yes, we prepared but didn't push any images yet. We'll propably do it some time in future.

**Comment by g-dx** [comment]
Any update on making renovate available in the AWS ECR public gallery?

**Comment by viceice** [comment]
not yet
